src/components/Passcode.vue

```diff
@@ -1,30 +1,41 @@
 <template>
-  <div class="inputs">
-    <div id="passcode" class="inputs d-flex flex-row justify-content-center mt-2">
-      <input
-        type="text"
-        id="input1"
-        v-on:keyup="inputenter(1)"
-        maxlength="1"
-      />
-      <input
-        v-on:keyup="inputenter(2)"
-        type="text"
-        id="input2"
-        maxlength="1"
-      />
-      <input
-        v-on:keyup="inputenter(3)"
-        type="text"
-        id="input3"
-        maxlength="1"
-      />
-      <input
-        v-on:keyup="inputenter(4)"
-        type="text"
-        id="input4"
-        maxlength="1"
-      />
+  <div class="list files">
+    <div class="log-in">
+      <h3>
+        Connect to production server
+        <input type="checkbox" v-on:click="productionServer" /><br />
+      </h3>
+      <div
+        id="passcode"
+        class="inputs d-flex flex-row justify-content-center mt-2"
+      >
+        <input
+          v-on:keyup="inputenter(0)"
+          type="text"
```

Comment:
How about `type="number"`
Then [remove the step buttons](https://www.w3schools.com/howto/howto_css_hide_arrow_number.asp) 

Would be nice to show an alert when a user types a character instead of a digit.